default.config.yml

```diff
@@ -1,8 +1,8 @@
 ---
 # Role: github
 # global git config settings
-git_email: "kziegler@chapman.edu"
-git_name: "Kayla Ziegler"
+git_email: " "
+git_name: " "
```

Comment:
Change these to something like `"TBA"` to make it a little clearer user needs to input their own values.